script/util.py

```diff
@@ -13,3 +14,23 @@ def valid_integration(integration):
         )
 
     return integration
+
+
+_MANIFEST_SORT_KEYS = {"domain": ".domain", "name": ".name"}
+
+
+def _sort_manifest_keys(key: str) -> str:
+    """Sort manifest keys."""
+    return _MANIFEST_SORT_KEYS.get(key, key)
+
+
+def sort_manifest(manifest: dict[str, Any]) -> bool:
+    """Sort manifest."""
+    keys = list(manifest.keys())
```

Comment:
```suggestion
    keys = list(manifest)
```

Reply:
Copied from the original, thanks for the simplification